code/game/machinery/camera/camera.dm

```diff
@@ -388,3 +388,18 @@
 	if(cameranet && get_turf(src) != prev_turf)
 		cameranet.updatePortableCamera(src)
 		prev_turf = get_turf(src)
+
+/obj/machinery/camera/get_vision_impairments(mob/user)
+	. = list()
+	if(view_range == short_range) //unfocused
+		. += global_hud.darkMask
```

Comment:
Looking through an unfocused camera has an actual visible effect for the player (same vision as welding helmet).